docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -240,33 +244,25 @@ version prior to 5.6, you must upgrade them after after installing
 Elasticsearch {version}.
 
 To get a list of the indices that need to be upgraded, install {xpack} and use
```

Comment:
Do we still want to tell people to "install xpack"?

Reply:
Good point, I've removed that.